src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java

```diff
@@ -537,6 +541,27 @@ private List<Position> decodeBinaryE(Channel channel, SocketAddress remoteAddres
                 int id = extension ? buf.readUnsignedShort() : buf.readUnsignedByte();
                 int length = buf.readUnsignedByte();
                 switch (id) {
+                    case 0x1D:
+                    case 0x1E:
+                    case 0x1F:
+                    case 0x20:
+                    case 0x21:
+                    case 0x22:
+                    case 0x23:
+                    case 0x24:
+                    case 0x25:
+                        String wifiMAC = ByteBufUtil.hexDump(buf.readSlice(6)).replaceAll("(..)", "$1:");
+                        int wifiRSSI = buf.readShortLE();
```

Comment:
1. I think with your code you will have trailing colon after the address.
2. Variables are not following our naming conventions. Once again I recommend inlining them. And if you do need, they should lower camel case. For example `wifiMac`.